src/main/java/com/mael/ttt/players/HumanPlayer.java

```diff
@@ -15,7 +15,7 @@ public HumanPlayer(UserInterface ui, Mark mark) {
     }
 
     public int getMove(Board board) {
-        String cellIndex = returnEmptyCellIndex(board, returnValidCellIndex());
+        String cellIndex = returnEmptyCellIndex(board, returnValidCellIndex(board));
```

Comment:
This line doesn't read very nice.
`returnEmptyCellIndex()` and a nested `returnValidCellIndex()` doesn't reveal the intend behind the method `getMove()` itself.
The `return...` at the beginning of the names don't really add any important value, do they?
Would something like `validCellIndexOf(board)` instead of `returnValidCellIndex(board)` help here?